### DISCUSSION TITLE
prometheus failed after configuring with service discovery

### DISCUSSION THREAD

**Discussion by vikramdevopsb79** [learndevopsonline/learndevopsonline#1914] [Q&A] [answered]
![Screenshot (442)](https://github.com/learndevopsonline/learndevopsonline/assets/169573156/b9aa481c-b5ce-4949-9cf3-87a5143ff887)

**Comment by chaithanya1812** [comment]
Hi @vikramdevopsb79 ,
## Issue: Attach  a role to prometheus server.

**Reply by vikramdevopsb79** [reply]
> Hi @vikramdevopsb79 ,
> 
> ## Issue: Attach a role to prometheus server.

already role is attached

**Reply by vikramdevopsb79** [reply, marked as answer]
issue solved in lab session
